hub/dev-environment/docker/overview.md

```diff
@@ -14,7 +14,7 @@ ms.date: 09/24/2020
 
 Using containers for remote development and deploying applications with the Docker platform is a very popular solution with many benefits. Learn more about the variety of support offered by Microsoft tools and services, including Windows Subsystem for Linux (WSL), Visual Studio, Visual Studio Code, .NET, and a broad variety of Azure services.
 
-## Docker on Windows 10
+## Docker on Windows 
```

Comment:
Whenever you remove the Windows version, be sure that the content applies to all versions of Windows, because the wording becomes inclusive of all Windows operating system versions.

Reply:
Agree with Aaron on this usage. Docker has a lot of complexity as a product and using just "Docker on Windows" here provides enough context for what it's pointing to. Thanks for brining up this point though Kristine.. it's a good one to keep in consideration.